plottr/gui/data_display.py

```diff
@@ -47,10 +47,10 @@ def _makeItem(self, name):
         ])
 
     @Slot(int)
-    def _processCbChange(self, _):
+    def _processCbChange(self, _: Any) -> None:
```

Comment:
according to the slot type it should be an integer
```suggestion
    def _processCbChange(self, _: int) -> None:
```

Reply:
Yes but it's also unused so ?

Reply:
yeah, unused inside the function, but from the outside i think the type should still be correct. in this case the "outside" is qt machinery of signals and slots, so it's not that important